### ISSUE TITLE
running salsify with preloaded videos

### ISSUE THREAD

**Issue by ushasigh** [excamera/alfalfa#93]
Hi,
I see that the salsify codebase contains a file fake-webcam.cc. Is there a way to run the the salsify system with saved videos in .y4m format instead of a real webcam.

**Comment by sadjad** [comment]
Yes! The easiest way to do this is to use `ffmpeg` to create a virtual webcam from a `.y4m` file (https://github.com/excamera/alfalfa/blob/master/scripts/fake-camera#L14), and then use that camera with Salsify (`--device` option). This approach requires `v4l2loopback` on Linux.
